tests/functional/test_model_completeness.py

```diff
@@ -46,3 +49,47 @@ def test_paginators_and_waiters_are_not_lost_in_new_version(
             raise AssertionError(
                 f"{type_name} must exist for {service_name}: {e}"
             )
+
+
+def _endpoint_rule_set_cases():
+    for service_name in available_services:
+        versions = loader.list_api_versions(service_name, 'service-2')
+        for version in versions:
+            yield service_name, version
+
+
+@pytest.mark.parametrize(
+    "service_name, version",
+    _endpoint_rule_set_cases(),
+)
+def test_all_endpoint_rule_sets_exist(service_name, version):
+    """Tests the existence of endpoint-rule-set.json for each service
+    and verifies that content is present."""
+    type_name = 'endpoint-rule-set'
+    data = loader.load_service_model(service_name, type_name, version)
+    assert len(data['rules']) >= 1
+
+
+def _endpoint_tests_cases():
+    for service_name in available_services:
+        yield service_name
```

Comment:
This looks redundant, can we just use `AVAILABLE_SERVICES` as input to parametrize?